### ISSUE TITLE
Cannot install with Mongoid version 7

### ISSUE THREAD

**Issue by infernalsirius** [PerfectMemory/mongoid-multitenancy#27]
```
Bundler could not find compatible versions for gem "mongoid":
  In snapshot (Gemfile.lock):
    mongoid (= 7.0.1)

  In Gemfile:
    mongoid (~> 7.0)

    mongoid-multitenancy (~> 2.0) was resolved to 2.0.1, which depends on
      mongoid (~> 6.0)
```

**Comment by Richard-Degenne** [comment]
Hi,

I'm not familiar enough with the gem to know about potential incompatibilities between `mongoid-multitenancy` and `mongoid 7`, but, for what it's worth, the unit tests are still passing after upgrading.

I'll create a PR, and discuss with @abrisse to see if we can release a new version.

**Comment by abrisse** [comment]
Fixed in d292cd2e93e7211e83620ed336d87b5daaf02849

**Release**: 2.0.2
